### DISCUSSION TITLE
Could not run Lighthouse

### DISCUSSION THREAD

**Discussion by skknb** [spatie/lighthouse-php#51] [Q&A]
i am using this package with Laravel, it was working fine but now it showing an error as following 

Spatie\Lighthouse\Exceptions\CouldNotRunLighthouse: Could not run Lighthouse. Error : `node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: connect ECONNREFUSED 127.0.0.1:34865
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1549:16) {
  errno: -111,
  code: &#039;ECONNREFUSED&#039;,
  syscall: &#039;connect&#039;,
  address: &#039;127.0.0.1&#039;,
  port: 34865
}

Node.js v18.19.0
` in file /opt/bitnami/apache/htdocs/vendor/spatie/lighthouse-php/src/Exceptions/CouldNotRunLighthouse.php on line 9

**Comment by NateWon** [comment]
Same here, not sure what has changed but was operational and no application related changes where made. Now is not working. 
I did update to the latest version of Ubuntu Desktop yesterday, but can't currently resolve the problem.

**Reply by adiologydev** [reply]
Same here, waiting for any resolution on this.

**Comment by captenmasin** [comment]
Having the exact same issue but **only** when running via Laravel jobs
Would love any resolution or comment on this, or if anyone has any alternative package recommendations?
